testing/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "snarkos-testing"
+version = "2.0.0"
+authors = [ "The Aleo Team <hello@aleo.org>" ]
+description = "A decentralized operating system"
+homepage = "https://aleo.org"
+repository = "https://github.com/AleoHQ/snarkOS"
+keywords = [
+    "aleo",
+    "cryptography",
+    "blockchain",
+    "decentralized",
+    "zero-knowledge"
+]
+categories = [ "cryptography", "operating-systems" ]
+license = "GPL-3.0"
+edition = "2018"
+
+[dependencies.snarkos-external-types]
+path = "../external-types"
+
+[dependencies.snarkvm]
+git = "https://github.com/AleoHQ/snarkVM.git"
+rev = "e7266f9"
+
+[dependencies.snarkos]
+path = ".."
+
+[dependencies.async-trait]
+version = "0.1"
+
+[dependencies.bincode]
+version = "1.0"
+
+[dependencies.pea2pea]
+version = "0.28"
+
+[dependencies.tokio]
+version = "1.0"
+features = ["macros", "rt-multi-thread", "time"]
+
+[dependencies.structopt]
+version = "0.3"
+
+[dependencies.tracing]
+version = "0.1"
+
+[dependencies.tracing-subscriber]
+version = "0.2"
```

Comment:
Why not `0.3` here?

Reply:
some copy-paste malfunction, it would seem :stuck_out_tongue:; will fix :+1: